src/sentry/static/sentry/app/components/duration.tsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function getDuration(seconds: number): string {
+  const value = Math.abs(seconds * 1000);
+  let result: number | string = '';
```

Comment:
Is it worth having another variable for the temporary results so that `result` can be `string`?

Reply:
yeah typing would be a bit more clear then huh

Reply:
ok just converted it to use `tn`